http_service/bugbug_http/worker.py

```diff
@@ -36,6 +36,11 @@ def main():
     )
     qs = sys.argv[1:] or ["default"]
     w = Worker(qs, connection=redis_conn)
+
+    # Write readiness probe file for Kubernetes
+    with open("/tmp/ready", "w") as f:
+        f.write("ready\n")
```

Comment:
nit: we don't even need to write something in it:
```suggestion
        pass
```